experiment/theory.md

```diff
@@ -1 +1,5 @@
-### Link your theory in here
\ No newline at end of file
+- In a impact test a specially prepared notched specimen is fractured by a single blow from a heavy hammer and energy required being a measure of resistance to impact. Impact load is produced by a swinging of an impact weight W (hammer) from a height h. 
```

Comment:
Why should these points be in a list?
Also check for grammatical mistakes here.